GoMeet/src/main/webapp/js/email-dao.js

```diff
@@ -0,0 +1,35 @@
+class EmailDAO {
+  /** Data Access Object for sending emails */
+
+  static endpoint = '/email'; // URL endpoint of the associated servlet
+  
+  /**
+   * Completes a POST request to send an email invitation with the 
+   * meeting event link to all addresses in the guest list 
+   * @param {String} meetingTimeId the unique entity key of the 
+   * MeetingEvent that the guest list is invited to 
+   * @param {List} guestList the list of guests' email addresses 
+   * @return JSON containing the sent status of the emails  
+   */
+  static async inviteGuestsToMeeting(meetingEventId, guestList) {
+    if (meetingEventId === null || meetingEventId === undefined) {
+      throw new Error(INSUFFICIENT_REQUEST_PARAM);
+    }
+
+    if (guestList === null || guestList === undefined) {
+      throw new Error(INSUFFICIENT_REQUEST_PARAM); 
+    }
+
+    if (typeof meetingEventId !== 'string' || typeof guestList !== 'object' || typeof guestList[1] !== 'string') {
```

Comment:
Why is the code only checking the second element of the guestList?

Reply:
Whoops this was meant to be guestList[0] but yes, I should check each element in the array. I was thinking it was a java array and assumed that each element would be of the same type.